### PR TITLE
[Update] - Change casing.

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

**Comment by niksedk** [comment]
What is the performance improvement? 1.5x, 3x or ?

File: src/Forms/ChangeCasing.cs [hunk 1]
```diff
@@ -65,13 +67,15 @@ internal void FixCasing(Subtitle subtitle, string language)
...
+            bool fixLowercaseI = language.StartsWith("en", StringComparison.Ordinal) && radioButtonNormal.Checked;
+
             string lastLine = string.Empty;
             foreach (Paragraph p in subtitle.Paragraphs)
             {
                 p.Text = FixCasing(p.Text, lastLine, namesEtc);
 
                 // fix casing of English alone i to I
-                if (radioButtonNormal.Checked && language.StartsWith("en", StringComparison.Ordinal))
+                if (fixLowercaseI)
```
Comment (ivandrofly) [inline src/Forms/ChangeCasing.cs]: Well, for every paragraph we are checking:
`(radioButtonNormal.Checked && language.StartsWith("en", StringComparison.Ordinal))`. there is always a impact on accessing a variable (ref type).

Comment (ivandrofly) [inline src/Forms/ChangeCasing.cs]: Ah no-no... "heavy" may be wrong word in this case I just mean the whole operation can be reduced by 1...

**Comment by niksedk** [comment]
If this is still relevant please add a new pr with info about why, what and unit tests

**Pull request closed without merging**